pages/docs/reference/ranges.md

```diff
@@ -11,9 +11,12 @@ Range expressions are formed with `rangeTo` functions that have the operator for
 Range is defined for any comparable type, but for integral primitive types it has an optimized implementation. Here are some examples of using ranges
 
 ``` kotlin
-if (i in 1..10) { // equivalent of 1 <= i && i <= 10
+if (i in 1..10)  // equivalent of i in [1, 10], 10 is included
```

Comment:
The word "equivalent" implies this is real Kotlin syntax which it isn't. Maybe plain english would be more appropriate.